doc/5_saving.md

```diff
@@ -189,6 +189,135 @@ val company = Company("DataStax", address)
 sc.parallelize(Seq(company)).saveToCassandra("test", "companies")
 ```
 
+## Skipping Columns and Avoiding Tombstones on Writes (Connector Version 1.6+ and Cassandra 2.2+)
```

Comment:
@RussellSpitzer can you apply code highlighting for code? Just enclose those snippets with \`\`\`scala ... \`\`\` or \`\`\`sql ... \`\`\`